### ISSUE TITLE
CAN'T MAKE POST UNLESS YOU JOIN AN EVENT (while there are active no events)

### ISSUE THREAD

**Issue by mikekasprzak** [ludumdare/ludumdare#1683]

**Comment by local-minimum** [comment]
Is the solution posting on your own user feed? Or rather some root "no event" feed?

I'm a bit concerned that it will become confusing rather quickly if it is important where you post.

**Comment by zwrawr** [comment]
Some discussion here #1545

**Comment by mikekasprzak** [comment]
> Is the solution posting on your own user feed?

This potentially needs a few things:

* user slug editing
* post parent editing
* creating other kinds of items besides games
* item parent editing (and restricting, so games created for events can't be moved)
* creating items under a user
    * items should include metadata that lets you create posts under them

99% of things people post about relate to a project. Ideally we should be able to add many kinds of projects to their account (we'll just need to figure out what types of projects we need to support).

This post came out of a request by someone that wanted to post about their post-compo game released for a Ludum Dare event from BEFORE the new website was working.

**Comment by ajayyy** [comment]
Another side effect of this issue is that people that play Ludum Dare games on YouTube haven't been able to advertise their google forms

**Comment by hendrikboom3** [comment]
I've been posting a few progress blog posts every few hours during ld45, and now suddenly it won't let me post any more.  What gives?

**Comment by mikekasprzak** [comment]
There's a limit of something like 100 posts per event (to curb spamming),
but unless you have a lot of false starts, that shouldn't be a problem. If
so, I would check your user page for some unpublished posts, and just edit
those. Once you click publish, it'll pop up in the feed.

**Comment by local-minimum** [comment]
@mikekasprzak if you have several drafts for several events, it is a bit hard to see for what event they are drafts. Because that's how it works right? Maybe that should be an issue too?

**Comment by mikekasprzak** [comment]
Yeah, I'm not super happy with how a lot of that is working in general.

I'm in the process of making more time for myself. In hindsight I choose
poorly with my dayjob as far as having time to work on anything outside of
work, buuut that's going to take me several months to sort out.

**Comment by local-minimum** [comment]
@mikekasprzak I still have a lot of time since I've got myself a part time IT job, and I lack projects to dedicate myself to, so whenever you feel ready, let me know and I come running.

**Comment by wheybags** [comment]
Am I correct that you can't post between events unless you were in the *most recent* event? I would like to post an update about my LD49 game (I'm doing a steam release), but I didn't join the most recent event so it seems like I can't make posts. It would be nice if I... could somehow :p
